Source/WPEFramework/PluginServer.h

```diff
@@ -1394,1571 +1416,1592 @@ namespace PluginHost {
+                        dataPath += callsign + '/';
+                        persistentPath += callsign + '/';
+                        volatilePath += callsign + '/';
                     }
-                    uint32_t Instantiate(
-                        const uint32_t requestId,
-                        const string& libraryName,
-                        const string& className,
-                        const string& callsign,
-                        const uint32_t interfaceId,
-                        const uint32_t version,
-                        const string& user,
-                        const string& group,
-                        const uint8_t threads,
-                        const int8_t priority,
-                        const string configuration) override
-                    {
-                        string persistentPath(_comms.PersistentPath());
-                        string dataPath(_comms.DataPath());
-                        string volatilePath(_comms.VolatilePath());
-
-                        if (callsign.empty() == false) {
-                            dataPath += callsign + '/';
-                            persistentPath += callsign + '/';
-                            volatilePath += callsign + '/';
-                        }
 
-                        uint32_t id;
-                        RPC::Config config(_connector, _comms.Application(), persistentPath, _comms.SystemPath(), dataPath, volatilePath, _comms.AppPath(), _comms.ProxyStubPath());
-                        RPC::Object instance(libraryName, className, callsign, interfaceId, version, user, group, threads, priority, RPC::Object::HostType::LOCAL, _T(""), configuration);
+                    uint32_t id;
+                    RPC::Config config(_connector, _comms.Application(), persistentPath, _comms.SystemPath(), dataPath, volatilePath, _comms.AppPath(), _comms.ProxyStubPath());
+                    RPC::Object instance(libraryName, className, callsign, interfaceId, version, user, group, threads, priority, RPC::Object::HostType::LOCAL, _T(""), configuration);
 
-                        RPC::Process process(requestId, config, instance);
+                    RPC::Process process(requestId, config, instance);
 
-                        return (process.Launch(id));
-                    }
+                    return (process.Launch(id));
+                }
 
-                    BEGIN_INTERFACE_MAP(RemoteInstantiation)
-                    INTERFACE_ENTRY(IRemoteInstantiation)
-                    END_INTERFACE_MAP
+                BEGIN_INTERFACE_MAP(RemoteInstantiation)
+                INTERFACE_ENTRY(IRemoteInstantiation)
+                END_INTERFACE_MAP
 
+            private:
+                mutable uint32_t _refCount;
+                ServiceMap& _parent;
+                const CommunicatorServer& _comms;
+                const string _connector;
+            };
+            class Override : public Core::JSON::Container {
+            private:
```

Comment:
Should be in the public section :)

Reply:
This is code writtten in the time that the MIPS compiler was not supporting std11 (and thus the delete keyword).
We wgreed that we would not actively start updating all the code, but only if we tocuh it. I do not think this code was tocuched so hence it is till prior to the std11 standard (private copy and assignment ocnstructors if you do not support them)

Reply:
I guess, it looks like I touched it because of the automatic styling tool I have installed. So I did not do it, my styling tool did it :-)